protoarray/src/main/java/tech/pegasys/teku/protoarray/ForkChoiceStrategy.java

```diff
@@ -452,10 +453,15 @@ void processBlock(
     return protoArray.getProtoNode(blockRoot);
   }
 
-  public void onExecutionPayloadResult(
-      final Bytes32 blockRoot,
-      final ExecutionPayloadStatus status,
-      final Optional<Bytes32> latestValidHash) {
+  public void onExecutionPayloadResult(final Bytes32 blockRoot, final ExecutePayloadResult result) {
+    if (result.hasFailedExecution()) {
+      LOG.warn(
+          "Payload for block root {} not processed, failed execution, EL is offline",
```

Comment:
What about: "Unable to Execute Payload for block root {}, Execution Engine is offline" ?

Reply:
I like it!